android/src/main/kotlin/live/hms/hmssdk_flutter/HmssdkFlutterPlugin.kt

```diff
@@ -985,4 +986,23 @@ class HmssdkFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware,
         getLocalPeer().videoTrack?.setMute(!(allowed!!))
         result?.success(null)
     }
+
+    private fun setVolume(call: MethodCall){
+        val peerId = call.argument<String>("peer_id")
```

Comment:
should be `track_Id` instead of `peer_Id`.
Peer reference should not be required, only the track object